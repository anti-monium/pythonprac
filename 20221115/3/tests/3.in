import time
start_time_slots = time.time()
alp = Alpha()
for i in range(100000):
	for c in alph:
		setattr(alp, c, i // ord(c))
s = 0
for i in range(100000):
	for c in alph:
		s += getattr(alp, c) % ord(c)
print(s)
time_slots = time.time() - start_time_slots

start_time_no_slots = time.time()
alq = AlphaQ()
for i in range(100000):
	for c in alph:
		setattr(alq, c, i // ord(c))
s = 0
for i in range(100000):
	for c in alph:
		s += getattr(alq, c) % ord(c)
print(s)
time_no_slots = time.time() - start_time_no_slots

print(time_slots < time_no_slots)
