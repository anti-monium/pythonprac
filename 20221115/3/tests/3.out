151400000
151400000
True
